src/value/owned.rs

```diff
@@ -467,111 +466,111 @@ impl PartialEq for Element {
 
 impl Eq for Element {}
 
-impl From<Value> for Element {
-    fn from(val: Value) -> Self {
-        Self::new(vec![], val)
```

Comment:
🗺️ Changes in this file were made to guarantee that Rust values with an obvious mapping to Ion `Element`s could be passed into APIs without needing explicit boilerplate conversion code.